packages/server/test/instruments.spec.ts

```
@@ -0,0 +1,33 @@
+      return {
+        instruments: {
+          request: async (_, wrapped) => {
+            results.push(`pre-${name}`);
+            await wrapped();
+            results.push(`post-${name}`);
+          },
+        },
+      };
+    }
+
+    const adapter = createServerAdapter<{}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)